docs/pages/desktop-access/discovery.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: Desktop Discover
+description: Automatically add Teleport Desktop Access to your infrastructure
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="11"
+>
+
+The features documented on this page are available in Teleport `11` and higher.
```

Comment:
I think it's 10.2 or higher. @ryanclark to confirm.

Reply:
10.2.6 or higher